content/client-lib-development-guide/features.textile

```diff
@@ -1494,6 +1531,7 @@ class ClientOptions:
   httpMaxRetryDuration: Duration default 15s // TO3l6
   maxMessageSize: Int default 65536 // TO3l8
   maxFrameSize: Int default 524288 // TO3l8
+  plugins: Dict<PluginType, Plugin> // TO3p
```

Comment:
Comment is incorrect. Should be `TO3o`.

Reply:
https://github.com/ably/docs/pull/796/commits/dfaed3d05b2eac3d71258ba37ac21b2864afddde